### PR TITLE
Fix/delft3 dgt 872 database view

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,13 +6,20 @@
...
       <strong>We're sorry but delft3d-gt-ui doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
     </noscript>
-    <!-- built files will be auto injected -->
-    <div id="app"></div>
+
+    <div id="app">
+
+
+    </div>
+    <script src="3dviewer/jsdap-8.1.0.min.js"></script>
```
Comment (SiggyF) [inline public/index.html:21]: find source

File: public/index.html [hunk 2]
```diff
@@ -6,13 +6,20 @@
...
+    <script src="3dviewer/Viewer3D.min.js"></script>
```
Comment (SiggyF) [inline public/index.html:22]: find source

File: src/components/ModelCard.vue [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
+    <div class="row">
+      <div class="col-xs-8 col-md-6">
+        <span v-if="model.data.shared == 'p'"><i class="fa fa-fw fa-user" aria-hidden="true"></i></span>
+        <span v-if="model.data.shared == 'c'"><i class="fa fa-fw fa-group" aria-hidden="true"></i></span>
+        <span v-if="model.data.shared == 'w'"><i class="fa fa-fw fa-globe" aria-hidden="true"></i></span>
+        &nbsp;{{model.data.name}}
+      </div>
+      <div class="col-xs-4 col-md-3 my-auto">
+        <span class="badge" :class="[
+                                     (model.statusLevel == 'Finished') ? 'badge-success' : '',
```
Comment (SiggyF) [inline src/components/ModelCard.vue]: move to computed.

File: src/components/ModelCard.vue [hunk 4]
```diff
@@ -31,32 +32,20 @@
...
-
-    </template>
-
-    <template v-else>
-
-      model missing
-
-    </template>
-
+      <div class="col-xs-1 col-md-1 my-auto">
```
Comment (SiggyF) [inline src/components/ModelCard.vue:22]: same code as other PR?

File: src/store/index.js [hunk 5]
```diff
@@ -41,11 +35,77 @@ export default new Vuex.Store({
...
+          })
+          .fail(function (jqXhr) {
+            reject(jqXhr)
+          })
+      })
+    },
+
+    startSync (store) {
+      // var x = 0
+      // var intervalID = setInterval(() => {
```
Comment (SiggyF) [inline src/store/index.js]: cleanup

**Review by SiggyF — APPROVED** [review]
same code  as other  PR?

File: src/components/ModelCard.vue (hunk 3 above)
Comment (CindyvdVries) [reply src/components/ModelCard.vue]: moved to methods

File: src/components/ModelCard.vue (hunk 4 above)
Comment (CindyvdVries) [reply src/components/ModelCard.vue:22]: see mail

**Pull request merged**